### ISSUE TITLE
Polyaxon Logger

### ISSUE THREAD

**Issue by Oodapow** [PyTorchLightning/pytorch-lightning#2399]
## 🚀 Feature
I would like to have a logger for the polyaxon platform. Similar to the way there is such a logger for MLFlow, WandB, tensorboard and such.

### Motivation

Polyaxon is a fast growing, large scale, open source ML platfrorm that I'v been using for a couple of months, it is similar to WandB but with a larger scope.

### Pitch

A new logger to be added that would let the user use polyaxon as an experiment management system with ease.

### Additional context

To learn more about polyaxon you can visit the github repo [HERE](https://github.com/polyaxon/polyaxon) or the website [HERE](https://polyaxon.com/)

**Comment by Borda** [comment]
@Oodapow looks interesting, does is support also offline mode?
cc: @PyTorchLightning/core-contributors

**Comment by justusschock** [comment]
@Borda I think we once said, that new loggers should first go into bolts and only go into pl if they proved to be stable. Also we probably want the authors of the package implement it, since ideally they'd also maintain it.

**Comment by Borda** [comment]
> Borda I think we once said, that new loggers should first go into bolts and only go into pl if they proved to be stable. Also we probably want the authors of the package implement it, since ideally they'd also maintain it.

yes, it would be Bolts in the first place, I was wondering just if it allows offline :]

**Comment by Oodapow** [comment]
> @Oodapow looks interesting, does is support also offline mode?
> cc: @PyTorchLightning/core-contributors

No default offline mode that i know as of yet, but the platform cat be installed as CE on any kube cluster (microk8s - for single node dev env)
